docs/user-guide/cli-ftpplugin.md

```diff
@@ -53,54 +61,54 @@ After you install the plug-in, create an FTP profile. An FTP profile is recommen
     ```
     zowe config set profiles.zftp.properties.port <port number>
     ```
-4. If using a insecure connection, set the secureFtp value to false:
+
+    - `<port number>`
+
+      Specifies the port number for the instance.
+4. If using an insecure connection, set the `secureFtp` value to `false`:
 
     ```
     zowe config set profiles.zftp.properties.secureFtp false
     ```
+    You can now use your profile when you issue commands in the zftp command group.
 
+#### Creating an FTP profile manually
 
-You can now use your profile when you issue commands in the zftp command group.
+1.  Install the z/OS FTP Plug-in for Zowe CLI.
 
-### Creating plug-in profiles using a command
+2. Browse to the directory `C:\Users\<username>\.zowe`.
 
-The following steps describe how to create a profile using the `zowe profiles create` command.
+3. Open the `zowe.config.json` configuration file using a text editor or IDE, such as Visual Studio Code or IntelliJ.
 
-1. Open a terminal window and issue the following command:
+    :::note
+    
+    If the file does not exist, issue the following command to create the configuration file:
     ```
-    zowe profiles create zftp  <profile_name> --host <host> --port <port> --user <user> --password <password>
+    zowe config init --gc
     ```
+    
+    :::
 
-    **`profile_name`:**
-
-    Specifies a name for your profile.
-
-    **`host`:**
-
-    Specifies the host name for the instance.
+4. Add code to the "profiles" section as shown in the following example: **[is this code correct? should this code include a `secureFtp` value?]**
```

Comment:
@adam-wolfe - please respond

Reply:
I would add `"secureFtp": true` and remove the line for regionName since that is specific to CICS.